server/server.js

```diff
@@ -1,523 +1,524 @@
+          'profile.mapRadius': 5,
+          'profile.locationString': locator
         }
-
-    });
-
-    Accounts.onCreateUser(function(options, user, err) {
-
-        // console.log(options, user, err);
-
-        if (options.profile) {
-
-            user.profile = options.profile;
-
-            // Images
-            var picturelrg = "http://graph.facebook.com/" + user.services.facebook.id + "/picture/?type=large";
-            var picturesm = "http://graph.facebook.com/" + user.services.facebook.id + "/picture/?type=small";
-
-            options.profile.picturelrg = picturelrg;
-            options.profile.picturesm = picturesm;
-
-            // Votes
-            options.profile.upvotes = 0;
-            options.profile.neutralvotes = 0;
-            options.profile.downvotes = 0;
-
-            // Sold
-            options.profile.amountsold = 0;
-            options.profile.amountbought = 0;
-
-            // Ratings
-            options.profile.sumrating = 0;
-            options.profile.satisfactionrating = 0;
-            options.profile.describedrating = 0;
-            options.profile.efficiencyrating = 0;
-            options.profile.friendlyrating = 0;
-            options.profile.pointscore = 0;
-            options.profile.reviewscount = 0;
-
-            // IP
-            // ip = response.ip;
-
-            // Saved
-            options.profile.saves = {};
-
-            // History
-            options.profile.history = {};
-
-            return user;
+      })
+    },
+
+    /*
+     * @summary Search Filter
+     * @locus Server
+     */
+    searchFilter: function(options) {
+      // console.log(options.Categories);
+
+      //   var selectorCat = {
+      //   category: options.Categories
+      // }
+
+      var selectorCity = {
+        city: 'Queens'
+      };
+
+      var selectorCategory = {
+        category: 'Apparel'
+      };
+
+      console.log(selectorCategory);
+
+      var query = Listing.find({
+        $and: [selectorCategory, selectorCity]
+      }).fetch();
+      console.log(query);
+      return query;
+    },
+    /*
+     * @summary Add offer
+     * @locus Server
+     */
+    addOffer: function(options) {
+      Offer.insert({
+        createdAt: new Date(),
+        listing_title: options.listing_title,
+        img: options.img,
+        price: options.price,
+        offerprice: options.offerprice,
+        date: options.date,
+        location: options.location,
+        listingId: options.listingId,
+        lat: options.lat,
+        lng: options.lng,
+        creator_id: options.creator_id,
+        payment: options.payment,
+        status: options.status
+      });
+    },
+    // For: Report, Add offer, Accept Offer, Badge, new Follower, TB system
+    // Destination is set in options previously
+    /*
+     * @summary Send Notification
+     * @locus Server
+     */
+    pulseNotify: function(options) {
+      Notification.insert({
+        listing_title: options.listing_title,
```

Comment:
Identifier 'listing_title' is not in camel case.<br>Too many errors. (66% scanned).